src/Commands/BannedKeywordsBag.ts

```diff
@@ -0,0 +1,203 @@
+/**
+ * A container for banned keywords for variable names.
+ */
+export class BannedKeywordsBag {
+    /**
+     * Banned keywords, keyed in alphabetical order.
+     */
+    private bannedKeyWords: { [i: string]: Boolean };
```

Comment:
Prefer `boolean` over `Boolean`. `Boolean` implies these are created with `Boolean()` instead of just using the literal. Those are different in JS.